test/arrayops.jl

```diff
@@ -1868,3 +1868,63 @@ end
         @test size(a) == size(b)
     end
 end
+
+isdefined(:TestHelpers) || include(joinpath(dirname(@__FILE__), "TestHelpers.jl"))
+using TestHelpers.OAs
+
+@testset "accumulate, accumulate!" begin
+
```

Comment:
empty lines as the first or last thing in an indented block are kind of superfluous, just take up space (also with L1913 and L1927, immediately preceding an `end`)